.gitignore

```diff
@@ -46,3 +46,7 @@ rpm/src/
 
 *.pyc
 test/acceptance/properties.json
+cmake_install.cmake
+CMakeFiles
+Makefile
```

Comment:
We have 86 CMAKE generated Makefiles in the system so far.
Not good to see them in "git status" ...